CONTRIBUTING.md

```diff
@@ -127,49 +129,48 @@ Make a PR modifying the following files with the right version:
 export const PACKAGE_VERSION = 'X.X.X'
 ```
 
+#### Github publish
+
 Once the changes are merged on `main`, you can publish the current draft release via the [GitHub interface](https://github.com/meilisearch/meilisearch-js/releases): on this page, click on `Edit` (related to the draft release) > update the description (be sure you apply [these recommendations](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md#writting-the-release-description)) > when you are ready, click on `Publish release`.
 
 GitHub Actions will be triggered and push the package to [npm](https://www.npmjs.com/package/meilisearch).
 
 #### Release a `beta` Version
 
-Here are the steps to release a beta version of this package:
+This package is able to create multiple types of betas:
+- A standard package beta, working on the latest version of Meilisearch.
+- A beta implementing the changes of a rc version of Meilisearch.
+- A beta implementing a specific feature `prototype` of Meilisearch.
+
+Here are the steps to release a beta version of this package depending on its type:
 
-- Create a new branch containing the "beta" changes with the following format `xxx-beta` where `xxx` explains the context.
+1. Create a new branch containing the changes with the correct name format following these rules:
+    - `package beta`: create a branch `beta/xx-xx` with the context of your beta.
```

Comment:
Is this still valid?

Reply:
Yes! Its a beta beta 😂 nothing to do with meilisearch.